### ISSUE TITLE
Lunar lander example - lander can go through ground

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/paper-land#81]
Reported today during design meeting, @BLFiedler noticed that the lander can sometimes fall through ground.

**Comment by jessegreenberg** [comment]
We saw this a few times while working with @terracoda today too. And it seemed to happen after the lander hit the ground a few times. I wonder if the ground is actually not static and is getting pushed down from the collision force of the lander or something.

Otherwise, I was thinking this could be a performance issue where the lander is moving fully across the ground in a single timestep. Could limit dt in this case.

**Comment by jessegreenberg** [comment]
OK I played with it for a few minutes and after ~2 minutes of letting the lander fall it just started to go right through the ground every single time. So its a consistent bug and not a performance problem.

**Comment by jessegreenberg** [comment]
I am seeing the lander get stuck, and I saw it fall through the ground at least once again since I last tested. Something strange seems to happen whenever I minimize the board tab or have it in the background. That might indicate a dt problem. Maybe a large dt throws the engine state somehow and it can't recover well?

**Comment by jessegreenberg** [comment]
Its happening much more frequently with large mass and high gravity.

**Comment by jessegreenberg** [comment]
OK, I have done a few things to try to fix this.

1) Limit the time delta so that it can't do incorrect physics for poor performance or when the window comes back from being minimized.
2) I added an option ccdSpeedThreshold to the Lander body which enables more accurate collision detection when the lander is moving at really fast speeds (should prevent it from tunneling through the ground body)
3) I made the ground body much much taller so that it is less likely to tunnel through.

I also havent' seen any sticking since these changes.

@BLFiedler back to you if you would like to try to reproduce or feel free to close.
